mbof/views.py

```diff
@@ -4,18 +4,18 @@
 from django.shortcuts import render, get_object_or_404
 from rest_framework import viewsets
 
-from .models import Message, User, Vote
+from .models import Event, User, Vote
 from .serializers import MessageSerializer, UserSerializer, VoteSerializer
 
 
 def index(request):
     return render(request, 'messages/index.html', {
-        'latestMessageList': Message.objects.order_by('-postingTime'),
+        'latestMessageList': Event.objects.order_by('-postingTime'),
     })
 
```

Comment:
I believe `index()`, and the following three methods (`detail()`, `results()`, and `vote()`) can be removed.  I think I added these to let me test the API before @gsilver implemented the UI.  Try commenting these out and see what breaks.  Really, all I/O from the Django app should be through the rest API, making these methods (and ***everything*** in the `templates` directory) aren't needed.